tests/test_cirq_device.py

```diff
@@ -142,217 +143,104 @@ def test_pre_apply(self, cirq_device_1_wire):
-
-        cirq_device_2_wires.pre_measure()
-
-        ops = list(cirq_device_2_wires.circuit.all_operations())
-
-        assert len(ops) == 1
-
-        assert np.allclose(ops[0]._gate._matrix, np.array(U), **tol)
-
-    def test_hermitian_error(self, cirq_device_3_wires):
-        """Tests that an error is raised for a three-qubit hermitian observable."""
-        A = np.eye(6)
-        cirq_device_3_wires._obs_queue = [qml.Hermitian(np.array(A), [0, 1, 2], do_queue=False)]
-
-        with pytest.raises(
-            qml.DeviceError,
-            match="Cirq only supports single-qubit and two-qubit unitary gates and thus only single-qubit and two-qubit Hermitian observables.",
-        ):
-            cirq_device_3_wires.pre_measure()
-
-    def test_hermitian_matrix_caching(self, cirq_device_1_wire, tol):
-        """Tests that the diagonalizations in pre_measure are properly cached."""
-
-        A = np.array([[0, 1], [-1, 0]])
-        U = np.array([[-1, 1], [1, 1]]) / math.sqrt(2)
-        w = np.array([-1, 1])
-
-        cirq_device_1_wire.reset()
-        cirq_device_1_wire._obs_queue = [qml.Hermitian(A, 0, do_queue=False)]
-
-        with patch("numpy.linalg.eigh", return_value=(w, U)) as mock:
-            cirq_device_1_wire.pre_measure()
-
-            assert mock.called
-
-            Hkey = list(cirq_device_1_wire._eigs.keys())[0]
-
-            assert np.allclose(cirq_device_1_wire._eigs[Hkey]["eigval"], w, **tol)
-            assert np.allclose(cirq_device_1_wire._eigs[Hkey]["eigvec"], U, **tol)
-
-        with patch("numpy.linalg.eigh", return_value=(w, U)) as mock:
-            cirq_device_1_wire.pre_measure()
-
-            assert not mock.called
-
-    # fmt: off
     @pytest.mark.parametrize(
-        "gate,par,expected_cirq_gates",
+        "gate,expected_cirq_gates",
         [
-            ("PauliX", [], [cirq.X]),
-            ("PauliY", [], [cirq.Y]),
-            ("PauliZ", [], [cirq.Z]),
-            ("PauliX.inv", [], [cirq.X ** -1]),
-            ("PauliY.inv", [], [cirq.Y ** -1]),
-            ("PauliZ.inv", [], [cirq.Z ** -1]),
-            ("Hadamard", [], [cirq.H]),
-            ("Hadamard.inv", [], [cirq.H ** -1]),
-            ("S", [], [cirq.S]),
-            ("S.inv", [], [cirq.S ** -1]),
-            ("PhaseShift", [1.4], [cirq.ZPowGate(exponent=1.4 / np.pi)]),
-            ("PhaseShift", [-1.2], [cirq.ZPowGate(exponent=-1.2 / np.pi)]),
-            ("PhaseShift", [2], [cirq.ZPowGate(exponent=2 / np.pi)]),
-            ("PhaseShift.inv", [1.4], [cirq.ZPowGate(exponent=-1.4 / np.pi)]),
-            ("PhaseShift.inv", [-1.2], [cirq.ZPowGate(exponent=1.2 / np.pi)]),
-            ("PhaseShift.inv", [2], [cirq.ZPowGate(exponent=-2 / np.pi)]),
-            ("RX", [1.4], [cirq.Rx(1.4)]),
-            ("RX", [-1.2], [cirq.Rx(-1.2)]),
-            ("RX", [2], [cirq.Rx(2)]),
-            ("RX.inv", [1.4], [cirq.Rx(-1.4)]),
-            ("RX.inv", [-1.2], [cirq.Rx(1.2)]),
-            ("RX.inv", [2], [cirq.Rx(-2)]),
```

Comment:
Perhaps keeping 1-1 rotations tests (e.g. RX, Rot and their inverses) could be nice

Reply:
They are still there, it is just misaligned due to the blacking